### PR TITLE
Add documentation examples for `consider-using-get`.

### PR DESCRIPTION
**Pull request by mbyrnepr2**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|    | :bug: Bug fix          |
|    | :sparkles: New feature |
|    | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Part of #5953

### CODE REVIEW

**Comment by matusvalo** [comment]
I think that example will be better, when the default value of `get()` is used. In this way, the purpose of error is more clear to the reader.

File: doc/data/messages/c/consider-using-get/bad.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+knights = {"gallahad": "the pure", "robin": "the brave"}
+
+if "gallahad" in knights:  # [consider-using-get]
+    DESCRIPTION = knights["gallahad"]
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/c/consider-using-get/bad.py:4]:
```suggestion
knights = {"Gallahad": "the pure", "Robin": "the brave"}

if "Gallahad" in knights:  # [consider-using-get]
    DESCRIPTION = knights["Gallahad"]
else:
    DESCRIPTION = ""
```

File: doc/data/messages/c/consider-using-get/good.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+knights = {"gallahad": "the pure", "robin": "the brave"}
+
+description = knights.get("gallahad")
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/c/consider-using-get/good.py:3]:
```suggestion
knights = {"Gallahad": "the pure", "Robin": "the brave"}

description = knights.get("Gallahad", "")
```

**Review by Pierre-Sassoulas — COMMENTED** [review]
Some suggestions based on @matusvalo review

**Comment by mbyrnepr2** [comment]
Thanks @matusvalo & @Pierre-Sassoulas. I agree with your comments.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**